module-generation/scrape.py

```diff
@@ -0,0 +1,67 @@
+import os
+import json
+import requests
+from github import Github
+
+indexDir = "./scrape-data/"
```

Comment:
this name is a bit ambiguous wrt the `Index` module

Reply:
done